### DISCUSSION TITLE
Issue with Onion install

### DISCUSSION THREAD

**Discussion by greenwithao** [OnionUI/Onion#553] [General]
I am trying to install Onion and ran into a issue. 
When I get to the grow your own onion section it shows the consoles, apps, and expert section but nothing is listed below. When it reboots it shows the onion screen for a second and goes dark then nothing. left for long while have to pull battery wont respond to power button.

did firmware update

stock card works fine
stock sd says version 202204190828 in device info
tried emptying card and putting .tmp_update back in but same.

EDIT Onion version is v3.10.5
MY miyoo is a 1.1 version.

Let me know what to try or if you need more information.

Thanks

**Comment by dwmccoy** [comment]
My initial thought is bad SDCARD.  Try a different card if possible.  If you have no other SDCARDS, reformat the one you have and try again.

**Reply by greenwithao** [reply]
I reformatted and ran a block check on sd card. was fine.

I did find the problem. something was wrong with my tmp_update folder. something went wrong with extract. was different file size from second attempt. Not sure how it happened but after i figured that out it started working. Really surprised that it worked at all. just enough of the files to barely work.

Thanks
